libs/damap/src/lib/components/dmp/people/people.component.ts

```diff
@@ -1,50 +1,66 @@
-import {Component, EventEmitter, Inject, Input, OnInit, Output} from '@angular/core';
-import {ContributorRole} from '../../../domain/enum/contributor-role.enum';
-import {UntypedFormArray, UntypedFormGroup} from '@angular/forms';
-import {Contributor} from '../../../domain/contributor';
-import {IdentifierType} from '../../../domain/enum/identifier-type.enum';
-import {Observable, Subject, switchMap} from 'rxjs';
-import {BackendService} from '../../../services/backend.service';
-import {debounceTime, distinctUntilChanged} from 'rxjs/operators';
-import {Dataset} from '../../../domain/dataset';
-import {MAT_DIALOG_DATA, MatDialog} from '@angular/material/dialog';
+import {
+  Component,
+  EventEmitter,
+  Inject,
+  Input,
+  OnInit,
+  Output,
+} from '@angular/core';
+import { ContributorRole } from '../../../domain/enum/contributor-role.enum';
+import { UntypedFormArray, UntypedFormGroup } from '@angular/forms';
+import { Contributor } from '../../../domain/contributor';
+import { IdentifierType } from '../../../domain/enum/identifier-type.enum';
+import { Observable, Subject, switchMap } from 'rxjs';
+import { BackendService } from '../../../services/backend.service';
+import { debounceTime, distinctUntilChanged } from 'rxjs/operators';
+import { Dataset } from '../../../domain/dataset';
+import { MAT_DIALOG_DATA, MatDialog } from '@angular/material/dialog';
+import { ServiceType } from '../../../domain/enum/search-service-type.enum';
 
 @Component({
   selector: 'app-dmp-people',
   templateUrl: './people.component.html',
-  styleUrls: ['./people.component.css']
+  styleUrls: ['./people.component.css'],
 })
 export class PeopleComponent implements OnInit {
-
   @Input() projectMembers: Contributor[];
-
-  readonly roles: any = ContributorRole;
-  readonly identifierType = IdentifierType;
-
-  readonly translateEnumPrefix = 'enum.contributor.role.'
```

Comment:
Either keep that variable or replace it in the template with it's value.

Reply:
Not sure here, is not part of this issue. I think if you take this out will make issues on the html since seems to be used for adding the person role.

Reply:
Yes, it's been removed, meaining roles aren't displayed correctly in the dropdown. So `translateEnumPrefix` should be kept :)